src/components/Search/engines/StopFinder.js

```diff
@@ -3,13 +3,19 @@ import React from 'react';
 import Engine from './Engine';
 
 class StopFinder extends Engine {
-  constructor(endpoint = 'https://api.geops.io/stops/v1/') {
+  constructor(endpoint = 'https://api.geops.io/stops/v1/', options) {
     super();
+    this.params = options || {};
```

Comment:
```suggestion
    this.options = options;
```

Renaming the function parameter seems confusing to me.